apps/filebrowser/src/filebrowser/conf.py

```diff
@@ -40,6 +40,18 @@
   default=None,
   type=str)
 
+FILE_UPLOAD_CHUNK_SIZE = Config(
+  key="file_upload_chunk_size",
+  default=5000000,
+  type=int,
+  help=_('flag to change the chunk size of file upload'))
+
+CONCURRENT_MAX_CONNECTIONS = Config(
+  key="concurrent_max_connections",
+  default=5,
+  type=int,
+  help=_('flag to set the maximum number of conncurrent connections to upload file'))
```

Comment:
Improve help message and I think no need to mention flag. For reference, check help statements of other configs.

Reply:
changed it.